src/components/components/ebay-dialog-base/component.js

```diff
@@ -155,13 +184,13 @@ module.exports = {
                     this.emit('open');
                 } else {
                     this._triggerBodyScroll(false);
-                    const activeElement = document.activeElement;
+                    const activeElement = this.getActiveElement();
```

Comment:
Since calling the function clears the click target, shouldn't we cache the call in this function call to avoid calling it more than once and getting different `activeElement`s?

Reply:
I removed that clearing of the activeElement for now. 
I don't think there's a need to clear it when we call it. We are removing the event when the dialog opens.